capstone/src/main/java/com/google/univiz/servlets/SearchService.java

```diff
@@ -38,4 +40,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     List<SearchData> searchResults = searchResource.getSearchSuggestions(query);
     ServletHelper.writeJsonToResponse(gson, response, searchResults);
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String numberCollegesStr = request.getParameter("collegeNumber");
+    int numberOfColleges = Integer.parseInt(numberCollegesStr);
+    List<String> listToSend = new ArrayList<>();
+    for (int i = 0; i < numberOfColleges; i++) {
+      StringBuilder parameterStr = new StringBuilder();
+      parameterStr.append("college");
+      parameterStr.append(Integer.toString(i));
+      String collegeName = request.getParameter(parameterStr.toString());
+      listToSend.add(collegeName);
+    }
```

Comment:
What happens to the list after it's been created?

Reply:
It needs to be forwarded to CollegeVis and Maps servlets (trying to figure that out now haha)